docs/fsharp/tools/fsharp-interactive/index.md

```diff
@@ -229,13 +237,16 @@ The `#r` and `#load` directives seen previously are only available in F# Interac
 |Directive|Description|
 |---------|-----------|
 |`#r "nuget:..."`|References a package from NuGet|
+|`#r "extname:..."`|Reference a package from `extname` extension (such as `paket`)|
 |`#r "assembly-name.dll"`|References an assembly on disk|
 |`#load "file-name.fsx"`|Reads a source file, compiles it, and runs it.|
 |`#help`|Displays information about available directives.|
 |`#I`|Specifies an assembly search path in quotation marks.|
 |`#quit`|Terminates an F# Interactive session.|
 |`#time "on"` or `#time "off"`|By itself, `#time` toggles whether to display performance information. When it is `"on"`, F# Interactive measures real time, CPU time, and garbage collection information for each section of code that is interpreted and executed.|
 
+[^1]: More about [F# Interactive extensions](https://aka.ms/dotnetdepmanager).
```

Comment:
Can you add where you intended to insert this link?

Reply:
Sorry! Somehow, when I deal with filling the comment for the commit from github, I had to manage several windows and I think I changed the text in the other window.

It would be great if the commit comment dialog was movable and not graying the background, so it enables user to put the file in preview mode and review the modifications.

I fixed the adjustments you suggested, fixed a comma into a semicolumn, and added the footnote reference where intended.

Thanks for the review and adjustments, as always!